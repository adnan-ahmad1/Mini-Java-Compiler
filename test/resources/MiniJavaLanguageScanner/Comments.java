// you cant see me

/*
*
*
*
*/

/* ******* */

/*


 */

// method
public static void returnTrue(boolean a) {
    // comment
    return true;
}

// test test // test
