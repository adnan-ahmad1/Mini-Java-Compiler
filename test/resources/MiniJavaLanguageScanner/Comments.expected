PUBLIC STATIC VOID ID(returnTrue) LPAREN BOOLEAN ID(a) RPAREN LCURLY RETURN TRUE SEMICOLON RCURLY